docs/development/getting-started-locally.md

```diff
@@ -0,0 +1,137 @@
+# Etcd-Druid Local Setup
+
+This page aims to provide steps on how to setup Etcd-Druid locally with and without storage providers
+
+## Prerequisites 
+
+
+### [Optional] Installing [k9s](https://k9scli.io/topics/install/) 
+
+[k9s](https://k9scli.io/) is an interactive terminal based UI to play with your kubernetes cluster. Makes it easy to interact with the cluster resources
+
+On Mac OS & Linux, install using brew
+
+```sh
+# via Homebrew
+brew install k9s
+```
+
+On Windows,
+```sh
+# via scoop
+scoop install k9s
+# via chocolatey
+choco install k9s
+```
+
```

Comment:
I agree with @seshachalam-yv. It would be ideal to keep the external dependencies to a minimum for any guide. Simple `kubectl` commands should suffice for anyone who is running `etcd-druid` locally for the first time.

Reply:
> In this guide, `k9s` is not utilized on any of the steps. Given this, is it necessary to include it as a prerequisite?

Then can we move it out of prerequisite and put just optional heading for it? Or if it seems out of place then I can remove it.

Reply:
Thanks for the suggestion. I removed it in the latest commit.